### PR TITLE
FIX: Adjusting setting of ajax default header as not being passed for version check

### PR DESCRIPTION
**Pull request by blckmn**
The version check was occurring before the default ajax header was being set. This resulted in a "non-current" return, because the header was missing.

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]
* approving untested.

**Pull request merged**
